doc/plugins/lineprofiler.rst

```diff
@@ -0,0 +1,53 @@
+####################
+Spyder Line Profiler
+####################
+
+**Spyder-Line-Profiler** is a plugin to run the Python `line_profiler <https://pypi.python.org/pypi/line_profiler>`_.
+
+.. image:: /images/lineprofiler/lineprofiler-overview.png
+   :alt: Spyder with the Line Profiler pane open
+
+Using the terminal inside Spyder allows you to use the system shell rather than just the IPython console. You can use it to issue commands, interact
```

Comment:
```suggestion
The line_profiler will profile the time that individual lines of code take to execute.
```
This was probably from the other PR and I don't think it applies. Maybe here it would be worth describing what a line profiler does. I added that part from the link you included but feel free to change it.

Reply:
Thank you for pointing this out @juanis2112! Yes, I did copy the `.rst` from my previous PR for this, but overlooked this.